.github/workflows/wheels.yml

```diff
@@ -30,7 +30,80 @@ env:
   FORCE_COLOR: 1
 
 jobs:
-  build:
+  build-1-QEMU-emulated-wheels:
+    name: QEMU ${{ matrix.python-version }} ${{ matrix.spec }}
```

Comment:
Perhaps a more helpful piece of information in the list is not that this uses QEMU, but that it is aarch64? It would be more consistent with our listing of the architecture for the other jobs.
![Screenshot 2024-01-04 at 11 23 12 am](https://github.com/python-pillow/Pillow/assets/3112309/18f13c37-c6e5-4e04-87e8-606842540566)

Reply:
Sure, so like:

"aarch64 cp312 musllinux"

or

"aarch64 musllinux cp312"?

Reply:
Updated to use "aarch64 cp312 musllinux".

Reply:
Thanks. I think that's probably my preference also.